AlpsSDK/NotificationManager.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  NotificationManager.swift
+//  AlpsSDK
+//
+//  Created by Wen on 13.10.17.
+//  Copyright © 2017 Alps. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import UserNotifications
+
+class NotificationManager : NSObject, UNUserNotificationCenterDelegate {
+    var alpsManager : AlpsManager
+    var deviceToken : Data? = nil
+    var deviceTokenString : String = ""
```

Comment:
`var deviceTokenString = ""` is shorter to read. swift can figure out the type itself.